.gitignore

```diff
@@ -3,5 +3,4 @@ npm-debug.log*
 build
 node_modules
 coverage
-./reports/*
```

Comment:
You can use 
```
reports/*
!reports/*.keep
```
That works locally for me.

Reply:
Only because the .keep was not changed.